### PR TITLE
Support altering imports with Q Queries

### PR DESCRIPTION
**Pull request by deviantintegral**
This will need to be updated once https://github.com/Lullabot/mpx-php/pull/129 is merged and a new tag is out.

### CODE REVIEW

**Comment by juanolalla** [comment]
Oops, unit-kernel tests are failing

**Comment by deviantintegral** [comment]
Fixed @juanolalla !

File: src/Commands/MpxImporter.php [hunk 1]
```diff
@@ -157,10 +157,10 @@ public function warmCache(string $media_type_id) {
   private function selectAll(MediaTypeInterface $media_type): ObjectListIterator {
     $media_source = DataObjectImporter::loadMediaSource($media_type);
     $factory = $this->dataObjectFactoryCreator->fromMediaSource($media_source);
-    $fields = new ByFields();
-    $event = new ImportSelectEvent($fields, $media_source);
+    $query = new ObjectListQuery();
+    $event = new ImportSelectEvent($query, $media_source);
     $this->eventDispatcher->dispatch(ImportSelectEvent::IMPORT_SELECT, $event);
-    $results = $factory->select($fields, $media_source->getAccount());
+    $results = $factory->select($query, $media_source->getAccount());
```
Comment (juanolalla) [inline src/Commands/MpxImporter.php:163]: I see you're changing here the argument passed to $factory->select, but I can't see you had changed that in the select() definition.

**Review by juanolalla — APPROVED** [review]
Good work, just a question to understand it better.

File: src/Commands/MpxImporter.php (hunk 1 above)
Comment (deviantintegral) [reply src/Commands/MpxImporter.php:163]: Ah yes, `select()` is in the mpx-php library. This PR is addressing the API changes between the 0.3.0 and 0.4.0 releases. See https://github.com/Lullabot/mpx-php/pull/129/files

**Pull request merged**